wolfssl/openssl/asn1.h

```diff
@@ -93,6 +93,17 @@
 #define ASN1_STRING_FLAG_EMBED           0x080
 
 
+/* size limits specified in RFC2459 */
+# define ub_name                         32768
+# define ub_common_name                  64
```

Comment:
We should try to map these to the wolfCrypt `CTC_*_SIZE` counterparts.

Reply:
Agree

Reply:
I'm only seeing `CTC_*_SIZE ` defined in _wolfssl/wolfcrypt/asn_public.h_:
```
enum Ctc_Misc {
    CTC_COUNTRY_SIZE  =     2,
    CTC_NAME_SIZE     = WC_CTC_NAME_SIZE,
    CTC_DATE_SIZE     =    32,
    CTC_MAX_ALT_SIZE  = WC_CTC_MAX_ALT_SIZE, /* may be huge, default: 16384 */
    CTC_SERIAL_SIZE   =    20,
    CTC_GEN_SERIAL_SZ =    16,
#ifdef WOLFSSL_CERT_EXT
    /* AKID could contains: hash + (Option) AuthCertIssuer,AuthCertSerialNum
     * We support only hash */
    CTC_MAX_SKID_SIZE = 32, /* SHA256_DIGEST_SIZE */
    CTC_MAX_AKID_SIZE = 32, /* SHA256_DIGEST_SIZE */
    CTC_MAX_CERTPOL_SZ = 64,
    CTC_MAX_CERTPOL_NB = 2 /* Max number of Certificate Policy */
#endif /* WOLFSSL_CERT_EXT */
};
```

that can be directly mapped to `ub_common_name` (both are `64`). With the other ub_*, how should I proceed? Perhaps we can also expand `CTC_NAME_SIZE` to map to `ub_organization_name`, `ub_organization_unit_name` and `ub_title` which are also `64`?

Reply:
The other 64 byte values may be defined as `CTC_NAME_SIZE` since other places use that as the size of those fields. Other values that aren't defined in `asn_public.h` directly would require more investigation. It would be best to map these to the size that is used in `asn.c` and these sizes may not always line up with those defined by the RFC.

Reply:
@julek-wolfssl after deeper into `CTC_NAME_SIZE` and related code such as `CertName` and `SetNameFromDcert()`, I believe all these ub_* values can be set to be `CTC_NAME_SIZE`. I've opted to remove `ub_name` as it's utility appears low.

Reply:
This looks like the right way to go. Thanks!